app.js

```diff
@@ -22,14 +23,18 @@ const receiver = new ExpressReceiver({
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   clientId: process.env.SLACK_CLIENT_ID,
   clientSecret: process.env.SLACK_CLIENT_SECRET,
-  token: process.env.SLACK_BOT_TOKEN,
```

Comment:
no longer necessary because of all the auth stuff?

Reply:
Yes, it's not necessary now.